src/hdmf_zarr/backend.py

```diff
@@ -566,8 +584,16 @@ def __get_ref(self, ref_object):
         source = (builder.source
                   if (builder.source is not None and os.path.isdir(builder.source))
                   else self.source)
+
         # Make the source relative to the current file
-        source = os.path.relpath(os.path.abspath(source), start=self.abspath)
+        # TODO: This check assumes that all links are internal links on export.
```

Comment:
Just FYI, looking through issues, we already have an open issue on export with external links here https://github.com/hdmf-dev/hdmf-zarr/issues/49  I.e., any specific issues we identify here related to export with external links, we should document there. I added a note, about this particular change in `__get_ref` to the issue, but if there are other items that you identified in this PR, then please add notes to that issue.